apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nav-sidebar/src/components/nav-sidebar/index.tsx

```diff
@@ -213,16 +217,17 @@ function WpcomBlockEditorNavSidebar() {
 					) }
 				</div>
 				<Button
+					aria-label={ __( 'View all pages in Dashboard', 'full-site-editing' ) }
```

Comment:
Sorry, another nitpick (yes, I'm that person! :) ) 

I think we need a definite article in front of 'Dashboard'.

`'Return to the Dashboard to view all pages'`

Also, if the user is viewing posts, there might have to be a version with 'posts' too, e.g., 

`'Return to the Dashboard to view all posts'`

I think the originals came from here: https://github.com/Automattic/wp-calypso/blob/master/apps/editing-toolkit/editing-toolkit-plugin/dotcom-fse/class-full-site-editing.php#L254

Not entirely sure what the best way to handle this would be right now. I'll take a think and get back to you.

Reply:
Already with ya on this.  I like your wording better than the original....I'll make that change. :)

Reply:
I think using `postType.slug` would work. 

I see you've already done it! https://github.com/Automattic/wp-calypso/pull/45266/files#diff-c37965521890589efb26a1f37e05b6d1R154 🤣 

Phil raises a good point about the alternatives (Home, Themes)... I forgot about that!

Maybe we'll have to handle the translations on the [Calypso side of things](https://github.com/Automattic/wp-calypso/blob/master/client/state/selectors/get-editor-close-config.js). @p-jackson will have a good idea ;)

Reply:
What I tried here (https://github.com/Automattic/wp-calypso/pull/45318/files#diff-c37965521890589efb26a1f37e05b6d1R234) was to use the regular label plus a further description. Regardless of the label the description is always "Returns to the dashboard". So if you wait long enough in VoiceOver you'll here: "Link: all pages ... returns to the dashboard"

But yes there is _another_ way of writing an aria label for every case and putting them in Calypso, but I was hoping the shared `aria-description` would allow us to avoid going down that route 😬

Reply:
> > Hm, the plot thickens. The WPicon button in the sidebar is NOT THE SAME button element that exists in the block editor header. I'm digging to understand more about this.
> 
> @autumnfjeld no it's not! Just to be confusing 😅
> 
> If you think of the sidebar like a modal, then the (W) that closes the side would be like the (x) button in the top-right of the modal.

I am wondering what the  reason was to not just used a standard close (x). I'm sure there were specific reasons....tied to the pre-existence of the (W) in that top-left position.   My own new-user opinion is that the use of the (W) to open a sidebar is odd.  Especially when the sidebar open/close button has such an important logo (that, to me,  indicates nothing about a sidebar or editing navigation).   But this is an inquiry/discussion I'll park for now.

Reply:
Yeah I'm starting to think a more explicit close button would be nicer too tbh